CatLover.Api/services/breed.service.go

```diff
@@ -0,0 +1,71 @@
+package services
+
+import (
+	// "database/sql"
+	// _ "github.com/mattn/go-sqlite3"
+
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"catlover.api/models"
+)
+
+type BreedService interface {
+	GetAllBreeds() []models.Breed
+	GetOneBreed(id string) (models.Breed, int32)
+	OpenBreedJsonFile() []models.Breed
+}
+
+func GetAllBreeds() []models.Breed {
+
+	//If SQLite was working
+	// db, err := sql.Open("sqlite3", "../data/catlover.db")
+	// breeds, err := db.Query("SELECT * FROM Breed")
+	// if err != nil {
+	// 	panic(err)
+	// }
+	// breeds.Close()
+
+	var breeds = OpenBreedJsonFile()
+	return breeds
+}
+
+func GetOneBreed(id string) (models.Breed, int32) {
+
+	//If SQLite was working
+	// db, err := sql.Open("sqlite3", "../data/catlover.db")
+	// breeds, err := db.Query("SELECT * FROM Breed WHERE Id = " + id)
+	// if err != nil {
+	// 	panic(err)
+	// }
+	// breeds.Close()
+
+	var breeds = OpenBreedJsonFile()
+	for index := range breeds {
+		if breeds[index].Id == id {
+			return breeds[index], 1
+		}
+	}
+	var breed = models.Breed{}
+	return breed, 0
+}
+
+func OpenBreedJsonFile() []models.Breed {
```

Comment:
Yes. In my mind i wanted to create a public method "OpenJsonFile(string fileName)" in a service class, in order to use it from all the places, where i wanted to get data. I tried without success so i live it like this, because it was working.